### PR TITLE
Add allocated space metrics to oracledb charts

### LINKED ISSUE #10196: Oracle monitoring reports wrong data
##### Bug report summary

Started to configure Oracle monitoring today and got confused about the reported metrics, so I think this needs to be reported. When I use this SQL in my Oracle database

```
set colsep '|'
set echo off
set feedback off
set linesize 1000
set pagesize 0
set sqlprompt ''
set trimspool on
set headsep off

select nvl(b.tablespace_name,
nvl(a.tablespace_name,'UNKNOWN'))
tablespace_name,
kbytes_alloc kbytes,
kbytes_alloc-nvl(kbytes_free,0)
size_alloc_bytes,
round(((kbytes_alloc-nvl(kbytes_free,0))/ kbytes_alloc)*200) used_chart,
to_char(((kbytes_alloc-nvl(kbytes_free,0))/ kbytes_alloc)*100,
'999G999G999G999G999G999G990D00') ||'%' used,
data_files
from ( select sum(bytes)/1024/1024 Kbytes_free,
max(bytes)/1024/1024 largest,
tablespace_name
from  sys.dba_free_space
group by tablespace_name ) a,
( select sum(bytes)/1024/1024 Kbytes_alloc,
tablespace_name,
count(*) data_files
from sys.dba_data_files
group by tablespace_name ) b
where a.tablespace_name (+) = b.tablespace_name;
```

I get the sizes reported like this:

```
SYSTEM			      |       700|	  690.4375|	  197|				98.63%| 	1
SYSAUX			      |      1030|	   965.375|	  187|				93.73%| 	1
UNDOTBS1		      |      2420|	     30.25|	    3|				 1.25%| 	1
USERS			      |      57.5|	   53.8125|	  187|				93.59%| 	1
```

But the netdata plugin reports much lower values:

Screenshot from 2020-11-04 14-28-51

No idea where the difference is coming from. Can you help me with this?

##### OS / Environment

Ubuntu 18.04 LTS

##### Netdata version

1.26.0 

##### Component Name

Oracle

### PR DESCRIPTION
**Pull request by jurgenhaas**
Fixes #10196

### CODE REVIEW

File: collectors/python.d.plugin/oracledb/oracledb.chart.py [hunk 1]
```diff
@@ -711,3 +805,27 @@ def add_tablespace_to_charts(self, name):
                 1,
                 1000,
             ])
+        self.charts['allocated_size'].add_dimension(
+            [
+                '{0}_allocated_size'.format(name),
+                name,
+                'absolute',
+                1,
+                1024 * 1000,
```
Comment (ilyam8) [inline collectors/python.d.plugin/oracledb/oracledb.chart.py:814]:
```suggestion
                1000,
```

Comment (ilyam8) [reply collectors/python.d.plugin/oracledb/oracledb.chart.py:814]: allocated size chart units is `B`, don't need 1024

File: collectors/python.d.plugin/oracledb/oracledb.chart.py [hunk 2]
```diff
@@ -711,3 +805,27 @@ def add_tablespace_to_charts(self, name):
...
+                ])
+        self.charts['allocated_usage'].add_dimension(
+            [
+                '{0}_allocated_used'.format(name),
+                name,
+                'absolute',
+                1,
+                1024 * 1000,
```
Comment (ilyam8) [inline collectors/python.d.plugin/oracledb/oracledb.chart.py:822]:
```suggestion
                1000,
```

same

**Comment by OdysLam** [comment]
@jurgenhaas I see that this your first contribution to Netdata. Thank you so much for taking the time and welcome on-board!

**Comment by jurgenhaas** [comment]
Thanks @OdysLam for your kind words. I was surprised a bit because I started using NetData back in 2016, probably pretty early. And I had lots of discussions in issues and even "contributed" there over time as well. But you're right, this is the first code contribution though.

**Comment by OdysLam** [comment]
Hey @jurgenhaas, thanks for the info and loyalty. We would love to hear your story in the forums https://community.netdata.cloud/ ! 
Please do reach out if you have any concern or you need any help with further implementation :)

**Comment by ilyam8** [comment]
@jurgenhaas friendly ping!

**Comment by jurgenhaas** [comment]
Sorry @ilyam8 have you been waiting on me? I probably misunderstood your comments above as I was under the impression you've already changed my PR but obviously not, I had to confirm them. Missed that completely, sorry for the delay.

**Comment by ilyam8** [comment]
@jurgenhaas the code looks good to me :+1: Do you confirm that https://github.com/netdata/netdata/issues/10196 is fixed and you see correct values?

**Comment by jurgenhaas** [comment]
@ilyam8 yes, I can confirm that. It's working here on 6 instances - as expected.

**Review by ilyam8 — APPROVED** [review]

**Comment by ilyam8** [comment]
Many thanks!

**Comment by ilyam8** [comment]
@jurgenhaas last but not least - please add allocated space charts to [the collector readme](https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/oracledb)

**Review by vlvkobal — APPROVED** [review]

**Comment by jurgenhaas** [comment]
Added the missing piece to the readme file.

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**